openssl/src/x509/mod.rs

```diff
@@ -129,32 +129,34 @@ impl X509StoreContext {
 /// let cert_pem = cert.to_pem().unwrap();
 /// let pkey_pem = pkey.private_key_to_pem().unwrap();
 /// ```
-pub struct X509Generator {
+pub struct X509Generator<'ctx> {
```

Comment:
Adding this type parameter is unfortunately a breaking change. Maybe we should have `set_ca` take the cert and pkey by value and fix it when we cut the next breaking release?